WireNetwork/Sources/WireNetwork/APIs/Rest/UpdateEventsAPI/Event decoding/UpdateEventDecodingProxy+FeatureConfig.swift

```diff
@@ -85,6 +85,11 @@ extension UpdateEventDecodingProxy {
                 let event = FeatureConfigUpdateEvent(featureConfig: .selfDeletingMessages(config))
                 updateEvent = .featureConfig(.update(event))
 
+            case "assetAuditLog":
+                let config = try AssetAuditLogFeatureConfigDecoder().decode(from: container)
+                let event = FeatureConfigUpdateEvent(featureConfig: .assetAuditLog(config))
+                updateEvent = .featureConfig(.update(event))
+
```

Comment:
question: this I'd like to understand, this UpdateEventDecodingProxy+FeatureConfig it's to migrate events from legacy to new sync, no? If so why do we have to add new decoding, they will never get them